judgments/views/detail.py

```diff
@@ -80,8 +80,15 @@ def detail(request, judgment_uri):
 
     if judgment_uri.endswith("press-summary/1"):
         context["document_type"] = "press_summary"
+        context["linked_document_uri"] = judgment_uri.rstrip("press-summary/1")
     else:
         context["document_type"] = "judgment"
+        context["linked_document_uri"] = judgment_uri + "/press-summary/1"
```

Comment:
@dragon-dxw @jacksonj04 Eventually we will move this logic into api client, but either way, keeping the /1 with the possibility of /2 will mean we need to decide on what version we link to with the press summary button